### PR TITLE
Add generic user type

### PR DESCRIPTION
**Pull request by putskan**

### CODE REVIEW

File: server/server/users/models.py [hunk 1]
```diff
@@ -72,7 +82,7 @@ def get_queryset(self, *args, **kwargs):
             super()
             .get_queryset(*args, **kwargs)
             .filter(
-                user_type=User.Types.COORDINATOR,
+                user_type="COORDINATOR",
```
Comment (putskan) [inline server/server/users/models.py:85]: change to Coordiantor.__name__.upper()

File: server/server/utils/renderers.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+class GenericCSVRenderer(CSVRenderer):
+    results_field = "results"
+    fields = []
+
+    def delete_keys_from_dict(self, orig_dict, keys_whitelist):
+        """
+        recursively delete keys which are not in whitelist from dict
+        """
+        for k in list(orig_dict.keys()):
```
Comment (putskan) [inline server/server/utils/renderers.py:13]: change to original

File: server/server/utils/renderers.py [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+            if isinstance(v, dict):
+                self.delete_keys_from_dict(v, keys_whitelist)
+
+        return orig_dict
+
+    def render(self, data, media_type=None, renderer_context=None, writer_opts=None):
+        """
+        extract the array from results field & remove fields which were not specified in whitelist
+        """
+        if not self.fields:
```
Comment (putskan) [inline server/server/utils/renderers.py:27]: change to try except

**Review by thehiveril — APPROVED** [review]
🔥

**Pull request merged**